content/docs/2.0/scalers/cpu.md

```diff
@@ -20,7 +20,7 @@ triggers:
 - type: cpu
   metadata:
     # Required
-    type: Utilization/ AverageValue
+    type: Utilization # or AverageValue
```

Comment:
```suggestion
    type: Utilization # Allowed types are 'Utilization' or 'AverageValue'
```

Reply:
I've made this change as part of a separate PR, in case that helps: https://github.com/kedacore/keda-docs/pull/909/files